application/managers/http/schemas.py

```diff
@@ -0,0 +1,3 @@
+from pydantic import BaseModel
```

Comment:
This file is empty you can remove it. As I remember I had plans to keep all Pydantinc schemes  in `application/schemas`.